### PR TITLE
[Synapse] - update version suffix to beta

### PR DESCRIPTION
**Pull request by idear1203**

### CODE REVIEW

File: sdk/synapse/Azure.Analytics.Synapse.AccessControl/CHANGELOG.md [hunk 1]
```diff
@@ -1,6 +1,6 @@
 # Release History
 
-## 1.0.0-preview.3 (Unreleased)
+## 1.0.0-beta.3 (Unreleased)
```
Comment (aim-for-better) [inline sdk/synapse/Azure.Analytics.Synapse.AccessControl/CHANGELOG.md:3]: Do we need to change this to `2020-11-12` just like Azure.Analytics.Synapse.Artifacts/CHANGELOG.md

File: sdk/synapse/Azure.Analytics.Synapse.Spark/CHANGELOG.md [hunk 2]
```diff
@@ -1,6 +1,6 @@
 # Release History
 
-## 1.0.0-preview.3 (Unreleased)
+## 1.0.0-beta.3 (Unreleased)
```
Comment (aim-for-better) [inline sdk/synapse/Azure.Analytics.Synapse.Spark/CHANGELOG.md:3]: Do we need to change this to `2020-11-12` just like Azure.Analytics.Synapse.Artifacts/CHANGELOG.md and other component's CHANGELOG.md?

**Review by aim-for-better — APPROVED** [review]

File: sdk/synapse/Azure.Analytics.Synapse.AccessControl/CHANGELOG.md (hunk 1 above)
Comment (idear1203) [reply sdk/synapse/Azure.Analytics.Synapse.AccessControl/CHANGELOG.md:3]: No we don't release it this time.

File: sdk/synapse/Azure.Analytics.Synapse.Spark/CHANGELOG.md (hunk 2 above)
Comment (idear1203) [reply sdk/synapse/Azure.Analytics.Synapse.Spark/CHANGELOG.md:3]: No we don't release it this time.

**Pull request merged**